sched/Kconfig

```diff
@@ -420,6 +420,13 @@ config RR_INTERVAL
 		Round robin scheduling (SCHED_RR) is enabled by setting this
 		interval to a positive, non-zero value.
 
+config SCHED_RR_IDLETASK
```

Comment:
why need this option? it's always good to disable time slice on idle thread.

Reply:
Keep idle task compatible? Maybe someone depend on this behaving?

Reply:
But It's a wrong behavior, should be fixed. If we follow this direction, any fix need add an option to keep the old behavior, which just make the thing worse.

Reply:
Here has some discussion about Kconfig:
https://www.mail-archive.com/dev@nuttx.apache.org/msg07946.html